android/SherpaOnnxTtsEngine/app/src/main/java/com/k2fsa/sherpa/onnx/tts/engine/TtsService.kt

```
@@ -110,13 +110,31 @@ class TtsService : TextToSpeechService() {
         val country = request.country
         val variant = request.variant
         val text = request.charSequenceText.toString()
+        // Map Android TTS speech rate (where 100 == normal) to engine speed (1.0 == normal)
+        // Allow per-request override from external apps; fallback to engine default if absent.
+        val engineSpeed: Float = run {
+            val rate = try {
+                request.speechRate
+            } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)